deisctl/units/deis-mesos-master.service

```diff
@@ -13,7 +13,7 @@ ExecStartPre=-/usr/bin/docker rm deis-mesos-master
 ExecStartPre=/bin/sh -c "docker inspect deis-mesos-master-data >/dev/null 2>&1 || docker run --name deis-mesos-master-data -v /tmp/mesos-master alpine:3.1 /bin/true"
 ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/mesos-master` && docker history $IMAGE >/dev/null 2>&1 || docker pull $IMAGE"
 ExecStart=/usr/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/mesos-master` && docker run --volumes-from=deis-mesos-master-data --name=deis-mesos-master --privileged --net=host -e HOST=$COREOS_PRIVATE_IPV4 $IMAGE"
-ExecStop=-/usr/bin/docker stop deis-mesos-master
+ExecStop=/usr/bin/docker stop deis-mesos-master
```

Comment:
I think the intention of prefixing with `-` here is that if the container is stopped by the time the `ExecStop` runs, the entire unit won't fail. I think that's something we want. Without this change, if the container is stopped, the `ExecStop` will run and fail.